icons/shield_glct_lhct.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+   <path d="M2,1.42108547e-14 L18,1.42108547e-14 C19.1045695,1.40079486e-14 20,0.8954305 20,2 L20,18 C20,19.1045695 19.1045695,20 18,20 L2,20 C0.8954305,20 -5.51538359e-14,19.1045695 -5.5067062e-14,18 L-5.5067062e-14,2 C-5.52023328e-14,0.8954305 0.8954305,1.41917162e-14 2,1.42108547e-14 Z" fill="#006747" fill-rule="nonzero"></path>
```

Comment:
Where possible, we’ve been leaving lines as lines rather than converting their strokes to filled paths. For instance, many shields implement the outer border as a rectangle or circle with a stroke and no fill. It all gets rasterized at build time, so maintainability is more important than optimizing rendering performance.

Reply:
Sounds good. My head was still in temaki icon design world where we want file size as small as possible with no strokes.